<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Functionally characterise a list of genes — characteriseGeneset • vissE</title><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/flatly/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><meta property="og:title" content="Functionally characterise a list of genes — characteriseGeneset"><meta property="og:description" content="This function can be used to perform a network-based enrichment analysis of a
list of genes. The list of genes are characterised based on their similarity
with gene sets from the MSigDB. A network of similar gene sets is retrieved
using this function."><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body data-spy="scroll" data-target="#toc">
    

    <div class="container template-reference-topic">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">vissE</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.1.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../articles/vissE.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul><ul class="nav navbar-nav navbar-right"><li>
  <a href="https://github.com/DavisLaboratory/vissE">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul></div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Functionally characterise a list of genes</h1>
    <small class="dont-index">Source: <a href="https://github.com/DavisLaboratory/vissE/blob/master/R/characteriseGeneset.R"><code>R/characteriseGeneset.R</code></a></small>
    <div class="hidden name"><code>characteriseGeneset.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>This function can be used to perform a network-based enrichment analysis of a
list of genes. The list of genes are characterised based on their similarity
with gene sets from the MSigDB. A network of similar gene sets is retrieved
using this function.</p>
    </div>

    <div class="ref-usage sourceCode"><pre class="sourceCode r"><code><span class="fu">characteriseGeneset</span><span class="op">(</span>
  <span class="va">gs</span>,
  thresh <span class="op">=</span> <span class="fl">0.2</span>,
  measure <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"ovlapcoef"</span>, <span class="st">"jaccard"</span><span class="op">)</span>,
  gscolcs <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="st">"h"</span>, <span class="st">"c2"</span>, <span class="st">"c5"</span><span class="op">)</span>
<span class="op">)</span></code></pre></div>

    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a>Arguments</h2>
    <table class="ref-arguments"><colgroup><col class="name"><col class="desc"></colgroup><tr><th>gs</th>
      <td><p>a GeneSet object, representing the list of genes that need to be
characterised.</p></td>
    </tr><tr><th>thresh</th>
      <td><p>a numeric, specifying the threshold to discard pairs of gene
sets.</p></td>
    </tr><tr><th>measure</th>
      <td><p>a character, specifying the similarity measure to use:
<code>jaccard</code> for the Jaccard Index and <code>ovlapcoef</code> for the Overlap
Coefficient.</p></td>
    </tr><tr><th>gscolcs</th>
      <td><p>a character, listing the MSigDB collections to use as a
background (defaults to h, c2, and c5). Collection types can be retrieved
using <code><a href="https://rdrr.io/pkg/msigdb/man/listCollections.html">msigdb::listCollections()</a></code>.</p></td>
    </tr></table><h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>

    <p>an igraph object, containing gene sets that are similar to the query
set. The network contains relationships between results of the query too.</p>

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <div class="ref-examples sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va">GSEABase</span><span class="op">)</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Loading required package: BiocGenerics</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Attaching package: ‘BiocGenerics’</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> The following objects are masked from ‘package:stats’:</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span>     IQR, mad, sd, var, xtabs</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> The following objects are masked from ‘package:base’:</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span>     Filter, Find, Map, Position, Reduce, anyDuplicated, append,</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span>     as.data.frame, basename, cbind, colnames, dirname, do.call,</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span>     duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted,</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span>     lapply, mapply, match, mget, order, paste, pmax, pmax.int, pmin,</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span>     pmin.int, rank, rbind, rownames, sapply, setdiff, sort, table,</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span>     tapply, union, unique, unsplit, which.max, which.min</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Loading required package: Biobase</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Welcome to Bioconductor</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span>     Vignettes contain introductory material; view with</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span>     'browseVignettes()'. To cite Bioconductor, see</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span>     'citation("Biobase")', and for packages 'citation("pkgname")'.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Loading required package: annotate</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Loading required package: AnnotationDbi</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Loading required package: stats4</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Loading required package: IRanges</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Loading required package: S4Vectors</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Attaching package: ‘S4Vectors’</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> The following objects are masked from ‘package:base’:</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span>     I, expand.grid, unname</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Loading required package: XML</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Loading required package: graph</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Attaching package: ‘graph’</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> The following object is masked from ‘package:XML’:</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span>     addNode</span>
<span class="r-in"><span class="fu"><a href="https://rdrr.io/r/utils/data.html">data</a></span><span class="op">(</span><span class="va">hgsc</span><span class="op">)</span></span>
<span class="r-in"></span>
<span class="r-in"><span class="co">#create a geneset using one of the Hallmark gene sets</span></span>
<span class="r-in"><span class="va">mySet</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/GSEABase/man/GeneSet-methods.html">GeneSet</a></span><span class="op">(</span></span>
<span class="r-in">  <span class="fu"><a href="https://rdrr.io/pkg/GSEABase/man/GeneSet-class.html">geneIds</a></span><span class="op">(</span><span class="va">hgsc</span><span class="op">[[</span><span class="fl">2</span><span class="op">]</span><span class="op">]</span><span class="op">)</span>,</span>
<span class="r-in">  setName <span class="op">=</span> <span class="st">'MySet'</span>,</span>
<span class="r-in">  geneIdType <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/GSEABase/man/GeneIdentifierType-constructors.html">SymbolIdentifier</a></span><span class="op">(</span><span class="op">)</span></span>
<span class="r-in"><span class="op">)</span></span>
<span class="r-in"></span>
<span class="r-in"><span class="co"># \donttest{</span></span>
<span class="r-in"><span class="co">#characterise the custom gene set</span></span>
<span class="r-in"><span class="va">ig</span> <span class="op">&lt;-</span> <span class="fu">characteriseGeneset</span><span class="op">(</span><span class="va">mySet</span><span class="op">)</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> snapshotDate(): 2021-10-06</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> see ?msigdb and browseVignettes('msigdb') for documentation</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> loading from cache</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'One hallmark of ECM proteins is their domain-based structure. Exploiting this characteristic, we established a list of diagnostic InterPro domains commonly found in ECM proteins. This domain list was used to screen the UniProt protein database. We know that some of the domains used to select positively for ECM proteins are also found in transmembrane receptors and proteins involved in cell adhesion (growth factor receptors, integrins, etc) that do not belong to the ECM. These families of proteins also display a subset of specific domains and transmembrane domains incompatible with definition as “extracellular matrix” proteins. Therefore, a second step comprised a negative selection using another set of domains and a transmembrane domain prediction. Manual curation of the matrisome lists also allowed us to add a very few known ECM proteins that do not contain any known domains. Protein-centric predictions were then converted to gene-centric lists. Finally, knowledge-ba [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'One hallmark of ECM proteins is their domain-based structure. Exploiting this characteristic, we established a list of diagnostic InterPro domains commonly found in ECM proteins. This domain list was used to screen the UniProt protein database. We know that some of the domains used to select positively for ECM proteins are also found in transmembrane receptors and proteins involved in cell adhesion (growth factor receptors, integrins, etc) that do not belong to the ECM. These families of proteins also display a subset of specific domains and transmembrane domains incompatible with definition as “extracellular matrix” proteins. Therefore, a second step comprised a negative selection using another set of domains and a transmembrane domain prediction. Manual curation of the matrisome lists also allowed us to add a very few known ECM proteins that do not contain any known domains. Protein-centric predictions were then converted to gene-centric lists. Finally, knowledge-ba [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'One hallmark of ECM proteins is their domain-based structure. Exploiting this characteristic, we established a list of diagnostic InterPro domains commonly found in ECM proteins. This domain list was used to screen the UniProt protein database. We know that some of the domains used to select positively for ECM proteins are also found in transmembrane receptors and proteins involved in cell adhesion (growth factor receptors, integrins, etc) that do not belong to the ECM. These families of proteins also display a subset of specific domains and transmembrane domains incompatible with definition as “extracellular matrix” proteins. Therefore, a second step comprised a negative selection using another set of domains and a transmembrane domain prediction. Manual curation of the matrisome lists also allowed us to add a very few known ECM proteins that do not contain any known domains. Protein-centric predictions were then converted to gene-centric lists. Finally, knowledge-ba [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'One hallmark of ECM proteins is their domain-based structure. Exploiting this characteristic, we established a list of diagnostic InterPro domains commonly found in ECM proteins. This domain list was used to screen the UniProt protein database. We know that some of the domains used to select positively for ECM proteins are also found in transmembrane receptors and proteins involved in cell adhesion (growth factor receptors, integrins, etc) that do not belong to the ECM. These families of proteins also display a subset of specific domains and transmembrane domains incompatible with definition as “extracellular matrix” proteins. Therefore, a second step comprised a negative selection using another set of domains and a transmembrane domain prediction. Manual curation of the matrisome lists also allowed us to add a very few known ECM proteins that do not contain any known domains. Protein-centric predictions were then converted to gene-centric lists. Finally, knowledge-ba [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'One hallmark of ECM proteins is their domain-based structure. Exploiting this characteristic, we established a list of diagnostic InterPro domains commonly found in ECM proteins. This domain list was used to screen the UniProt protein database. We know that some of the domains used to select positively for ECM proteins are also found in transmembrane receptors and proteins involved in cell adhesion (growth factor receptors, integrins, etc) that do not belong to the ECM. These families of proteins also display a subset of specific domains and transmembrane domains incompatible with definition as “extracellular matrix” proteins. Therefore, a second step comprised a negative selection using another set of domains and a transmembrane domain prediction. Manual curation of the matrisome lists also allowed us to add a very few known ECM proteins that do not contain any known domains. Protein-centric predictions were then converted to gene-centric lists. Finally, knowledge-ba [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'One hallmark of ECM proteins is their domain-based structure. Exploiting this characteristic, we established a list of diagnostic InterPro domains commonly found in ECM proteins. This domain list was used to screen the UniProt protein database. We know that some of the domains used to select positively for ECM proteins are also found in transmembrane receptors and proteins involved in cell adhesion (growth factor receptors, integrins, etc) that do not belong to the ECM. These families of proteins also display a subset of specific domains and transmembrane domains incompatible with definition as “extracellular matrix” proteins. Therefore, a second step comprised a negative selection using another set of domains and a transmembrane domain prediction. Manual curation of the matrisome lists also allowed us to add a very few known ECM proteins that do not contain any known domains. Protein-centric predictions were then converted to gene-centric lists. Finally, knowledge-ba [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'One hallmark of ECM proteins is their domain-based structure. Exploiting this characteristic, we established a list of diagnostic InterPro domains commonly found in ECM proteins. This domain list was used to screen the UniProt protein database. We know that some of the domains used to select positively for ECM proteins are also found in transmembrane receptors and proteins involved in cell adhesion (growth factor receptors, integrins, etc) that do not belong to the ECM. These families of proteins also display a subset of specific domains and transmembrane domains incompatible with definition as “extracellular matrix” proteins. Therefore, a second step comprised a negative selection using another set of domains and a transmembrane domain prediction. Manual curation of the matrisome lists also allowed us to add a very few known ECM proteins that do not contain any known domains. Protein-centric predictions were then converted to gene-centric lists. Finally, knowledge-ba [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'One hallmark of ECM proteins is their domain-based structure. Exploiting this characteristic, we established a list of diagnostic InterPro domains commonly found in ECM proteins. This domain list was used to screen the UniProt protein database. We know that some of the domains used to select positively for ECM proteins are also found in transmembrane receptors and proteins involved in cell adhesion (growth factor receptors, integrins, etc) that do not belong to the ECM. These families of proteins also display a subset of specific domains and transmembrane domains incompatible with definition as “extracellular matrix” proteins. Therefore, a second step comprised a negative selection using another set of domains and a transmembrane domain prediction. Manual curation of the matrisome lists also allowed us to add a very few known ECM proteins that do not contain any known domains. Protein-centric predictions were then converted to gene-centric lists. Finally, knowledge-ba [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'One hallmark of ECM proteins is their domain-based structure. Exploiting this characteristic, we established a list of diagnostic InterPro domains commonly found in ECM proteins. This domain list was used to screen the UniProt protein database. We know that some of the domains used to select positively for ECM proteins are also found in transmembrane receptors and proteins involved in cell adhesion (growth factor receptors, integrins, etc) that do not belong to the ECM. These families of proteins also display a subset of specific domains and transmembrane domains incompatible with definition as “extracellular matrix” proteins. Therefore, a second step comprised a negative selection using another set of domains and a transmembrane domain prediction. Manual curation of the matrisome lists also allowed us to add a very few known ECM proteins that do not contain any known domains. Protein-centric predictions were then converted to gene-centric lists. Finally, knowledge-ba [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'One hallmark of ECM proteins is their domain-based structure. Exploiting this characteristic, we established a list of diagnostic InterPro domains commonly found in ECM proteins. This domain list was used to screen the UniProt protein database. We know that some of the domains used to select positively for ECM proteins are also found in transmembrane receptors and proteins involved in cell adhesion (growth factor receptors, integrins, etc) that do not belong to the ECM. These families of proteins also display a subset of specific domains and transmembrane domains incompatible with definition as “extracellular matrix” proteins. Therefore, a second step comprised a negative selection using another set of domains and a transmembrane domain prediction. Manual curation of the matrisome lists also allowed us to add a very few known ECM proteins that do not contain any known domains. Protein-centric predictions were then converted to gene-centric lists. Finally, knowledge-ba [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'To elucidate the gene expression “footprint” of antigenically challenged T-cells which had been treated with anti-LFA-1, CTLA4Ig, anti-CD40-ligand antibodies, we performed microarray gene expression analysis comparing the expression profile of costimulatory blockade treated and untreated responder T-cells.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'To elucidate the gene expression “footprint” of antigenically challenged T-cells which had been treated with anti-LFA-1, CTLA4Ig, anti-CD40-ligand antibodies, we performed microarray gene expression analysis comparing the expression profile of costimulatory blockade treated and untreated responder T-cells.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'To elucidate the gene expression “footprint” of antigenically challenged T-cells which had been treated with anti-LFA-1, CTLA4Ig, anti-CD40-ligand antibodies, we performed microarray gene expression analysis comparing the expression profile of costimulatory blockade treated and untreated responder T-cells.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'To elucidate the gene expression “footprint” of antigenically challenged T-cells which had been treated with anti-LFA-1, CTLA4Ig, anti-CD40-ligand antibodies, we performed microarray gene expression analysis comparing the expression profile of costimulatory blockade treated and untreated responder T-cells.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'To elucidate the gene expression “footprint” of antigenically challenged T-cells which had been treated with anti-LFA-1, CTLA4Ig, anti-CD40-ligand antibodies, we performed microarray gene expression analysis comparing the expression profile of costimulatory blockade treated and untreated responder T-cells.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'To elucidate the gene expression “footprint” of antigenically challenged T-cells which had been treated with anti-LFA-1, CTLA4Ig, anti-CD40-ligand antibodies, we performed microarray gene expression analysis comparing the expression profile of costimulatory blockade treated and untreated responder T-cells.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'To elucidate the gene expression “footprint” of antigenically challenged T-cells which had been treated with anti-LFA-1, CTLA4Ig, anti-CD40-ligand antibodies, we performed microarray gene expression analysis comparing the expression profile of costimulatory blockade treated and untreated responder T-cells.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'To elucidate the gene expression “footprint” of antigenically challenged T-cells which had been treated with anti-LFA-1, CTLA4Ig, anti-CD40-ligand antibodies, we performed microarray gene expression analysis comparing the expression profile of costimulatory blockade treated and untreated responder T-cells.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'The process for making monocyte derived DCs (moDCs) has been previously described (46). All analysis was performed on “day 5” immature DCs. Etanercept 10mg/mL was added to experimental wells on days 0, 2, and 4. We chose this concentration of etanercept as it approximates the plasma concentration of drug when given 50mg BIW.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'The process for making monocyte derived DCs (moDCs) has been previously described (46). All analysis was performed on “day 5” immature DCs. Etanercept 10mg/mL was added to experimental wells on days 0, 2, and 4. We chose this concentration of etanercept as it approximates the plasma concentration of drug when given 50mg BIW.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'The objective of this study was to compare the transcriptional repertoire of mature human neutrophils before and after GM-CSF treatment by using oligonucleotide microarrays. Leukotriene B4 (LTB4) is an important pro-inflammatory lipid mediator generated by neutrophils upon activation. Granulocyte/macrophage colony-stimulating factor (GM-CSF) stimulation is known to enhance agonist-mediated LTB4 production of neutrophils within minutes, a process called “priming”. Here, we demonstrate that GM-CSF also limits the production of LTB4 by neutrophils via a transcriptional mechanism at later time points. We identified hematopoietic specific Ras homologous (RhoH)/translocation three four (TTF), which was induced following GM-CSF stimulation in neutrophils, as a key regulator in this process. Neutrophils derived from RhoH/TTF-deficient (Rhoh-/-) mice demonstrated increased LTB4 production upon activation compared with normal mouse neutrophils. Moreover, neutrophils from cystic [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'The objective of this study was to compare the transcriptional repertoire of mature human neutrophils before and after GM-CSF treatment by using oligonucleotide microarrays. Leukotriene B4 (LTB4) is an important pro-inflammatory lipid mediator generated by neutrophils upon activation. Granulocyte/macrophage colony-stimulating factor (GM-CSF) stimulation is known to enhance agonist-mediated LTB4 production of neutrophils within minutes, a process called “priming”. Here, we demonstrate that GM-CSF also limits the production of LTB4 by neutrophils via a transcriptional mechanism at later time points. We identified hematopoietic specific Ras homologous (RhoH)/translocation three four (TTF), which was induced following GM-CSF stimulation in neutrophils, as a key regulator in this process. Neutrophils derived from RhoH/TTF-deficient (Rhoh-/-) mice demonstrated increased LTB4 production upon activation compared with normal mouse neutrophils. Moreover, neutrophils from cystic [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Histone acetyltransferases (HATs) and deacetylases (HDACs) function antagonistically to control histone acetylation. As acetylation is a histone mark for active transcription, HATs have been associated with active and HDACs with inactive genes. We describe here genome-wide mapping of HATs and HDACs binding on chromatin and ﬁnd that both are found at active genes with acetylated histones. Our data provide evidence that HATs and HDACs are both targeted to transcribed regions of active genes by phosphorylated RNA Pol II. Furthermore, the majority of HDACs in the human genome function to reset chromatin by removing acetylation at active genes. Inactive genes that are primed by MLL-mediated histone H3K4 methylation are subject to a dynamic cycle of acetylation and deacetylation by transient HAT/HDAC binding, preventing Pol II from binding to these genes but poising them for future activation. Silent genes without any H3K4 methylation signal show no evidence of being bound by [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Histone acetyltransferases (HATs) and deacetylases (HDACs) function antagonistically to control histone acetylation. As acetylation is a histone mark for active transcription, HATs have been associated with active and HDACs with inactive genes. We describe here genome-wide mapping of HATs and HDACs binding on chromatin and ﬁnd that both are found at active genes with acetylated histones. Our data provide evidence that HATs and HDACs are both targeted to transcribed regions of active genes by phosphorylated RNA Pol II. Furthermore, the majority of HDACs in the human genome function to reset chromatin by removing acetylation at active genes. Inactive genes that are primed by MLL-mediated histone H3K4 methylation are subject to a dynamic cycle of acetylation and deacetylation by transient HAT/HDAC binding, preventing Pol II from binding to these genes but poising them for future activation. Silent genes without any H3K4 methylation signal show no evidence of being bound by [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Histone acetyltransferases (HATs) and deacetylases (HDACs) function antagonistically to control histone acetylation. As acetylation is a histone mark for active transcription, HATs have been associated with active and HDACs with inactive genes. We describe here genome-wide mapping of HATs and HDACs binding on chromatin and ﬁnd that both are found at active genes with acetylated histones. Our data provide evidence that HATs and HDACs are both targeted to transcribed regions of active genes by phosphorylated RNA Pol II. Furthermore, the majority of HDACs in the human genome function to reset chromatin by removing acetylation at active genes. Inactive genes that are primed by MLL-mediated histone H3K4 methylation are subject to a dynamic cycle of acetylation and deacetylation by transient HAT/HDAC binding, preventing Pol II from binding to these genes but poising them for future activation. Silent genes without any H3K4 methylation signal show no evidence of being bound by [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Histone acetyltransferases (HATs) and deacetylases (HDACs) function antagonistically to control histone acetylation. As acetylation is a histone mark for active transcription, HATs have been associated with active and HDACs with inactive genes. We describe here genome-wide mapping of HATs and HDACs binding on chromatin and ﬁnd that both are found at active genes with acetylated histones. Our data provide evidence that HATs and HDACs are both targeted to transcribed regions of active genes by phosphorylated RNA Pol II. Furthermore, the majority of HDACs in the human genome function to reset chromatin by removing acetylation at active genes. Inactive genes that are primed by MLL-mediated histone H3K4 methylation are subject to a dynamic cycle of acetylation and deacetylation by transient HAT/HDAC binding, preventing Pol II from binding to these genes but poising them for future activation. Silent genes without any H3K4 methylation signal show no evidence of being bound by [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Histone acetyltransferases (HATs) and deacetylases (HDACs) function antagonistically to control histone acetylation. As acetylation is a histone mark for active transcription, HATs have been associated with active and HDACs with inactive genes. We describe here genome-wide mapping of HATs and HDACs binding on chromatin and ﬁnd that both are found at active genes with acetylated histones. Our data provide evidence that HATs and HDACs are both targeted to transcribed regions of active genes by phosphorylated RNA Pol II. Furthermore, the majority of HDACs in the human genome function to reset chromatin by removing acetylation at active genes. Inactive genes that are primed by MLL-mediated histone H3K4 methylation are subject to a dynamic cycle of acetylation and deacetylation by transient HAT/HDAC binding, preventing Pol II from binding to these genes but poising them for future activation. Silent genes without any H3K4 methylation signal show no evidence of being bound by [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Histone acetyltransferases (HATs) and deacetylases (HDACs) function antagonistically to control histone acetylation. As acetylation is a histone mark for active transcription, HATs have been associated with active and HDACs with inactive genes. We describe here genome-wide mapping of HATs and HDACs binding on chromatin and ﬁnd that both are found at active genes with acetylated histones. Our data provide evidence that HATs and HDACs are both targeted to transcribed regions of active genes by phosphorylated RNA Pol II. Furthermore, the majority of HDACs in the human genome function to reset chromatin by removing acetylation at active genes. Inactive genes that are primed by MLL-mediated histone H3K4 methylation are subject to a dynamic cycle of acetylation and deacetylation by transient HAT/HDAC binding, preventing Pol II from binding to these genes but poising them for future activation. Silent genes without any H3K4 methylation signal show no evidence of being bound by [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'The interferon-producing plasmacytoid dendritic cells (PDC) share common progenitors with antigen-presenting classical dendritic cells (cDC), yet they possess distinct morphology and molecular features resembling those of lymphocytes. It is unclear whether the unique cell fate of PDC is actively maintained in the steady state. We report that the deletion of transcription factor E2-2 from mature peripheral PDC caused their spontaneous differentiation into cells with cDC properties. This included the loss of PDC markers, increase in MHC class II expression and T cell priming capacity, acquisition of dendritic morphology and induction of cDC signature genes. Genome-wide chromatin immunoprecipitation revealed direct binding of E2-2 to key PDC-specific and lymphoid genes, as well as to certain genes enriched in cDC. Thus, E2-2 actively maintains the cell fate of mature PDC and opposes the “default” cDC fate, in part through direct regulation of lineage-specific gene expres [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'The interferon-producing plasmacytoid dendritic cells (PDC) share common progenitors with antigen-presenting classical dendritic cells (cDC), yet they possess distinct morphology and molecular features resembling those of lymphocytes. It is unclear whether the unique cell fate of PDC is actively maintained in the steady state. We report that the deletion of transcription factor E2-2 from mature peripheral PDC caused their spontaneous differentiation into cells with cDC properties. This included the loss of PDC markers, increase in MHC class II expression and T cell priming capacity, acquisition of dendritic morphology and induction of cDC signature genes. Genome-wide chromatin immunoprecipitation revealed direct binding of E2-2 to key PDC-specific and lymphoid genes, as well as to certain genes enriched in cDC. Thus, E2-2 actively maintains the cell fate of mature PDC and opposes the “default” cDC fate, in part through direct regulation of lineage-specific gene expres [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'The interferon-producing plasmacytoid dendritic cells (PDC) share common progenitors with antigen-presenting classical dendritic cells (cDC), yet they possess distinct morphology and molecular features resembling those of lymphocytes. It is unclear whether the unique cell fate of PDC is actively maintained in the steady state. We report that the deletion of transcription factor E2-2 from mature peripheral PDC caused their spontaneous differentiation into cells with cDC properties. This included the loss of PDC markers, increase in MHC class II expression and T cell priming capacity, acquisition of dendritic morphology and induction of cDC signature genes. Genome-wide chromatin immunoprecipitation revealed direct binding of E2-2 to key PDC-specific and lymphoid genes, as well as to certain genes enriched in cDC. Thus, E2-2 actively maintains the cell fate of mature PDC and opposes the “default” cDC fate, in part through direct regulation of lineage-specific gene expres [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'The interferon-producing plasmacytoid dendritic cells (PDC) share common progenitors with antigen-presenting classical dendritic cells (cDC), yet they possess distinct morphology and molecular features resembling those of lymphocytes. It is unclear whether the unique cell fate of PDC is actively maintained in the steady state. We report that the deletion of transcription factor E2-2 from mature peripheral PDC caused their spontaneous differentiation into cells with cDC properties. This included the loss of PDC markers, increase in MHC class II expression and T cell priming capacity, acquisition of dendritic morphology and induction of cDC signature genes. Genome-wide chromatin immunoprecipitation revealed direct binding of E2-2 to key PDC-specific and lymphoid genes, as well as to certain genes enriched in cDC. Thus, E2-2 actively maintains the cell fate of mature PDC and opposes the “default” cDC fate, in part through direct regulation of lineage-specific gene expres [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'The interferon-producing plasmacytoid dendritic cells (PDC) share common progenitors with antigen-presenting classical dendritic cells (cDC), yet they possess distinct morphology and molecular features resembling those of lymphocytes. It is unclear whether the unique cell fate of PDC is actively maintained in the steady state. We report that the deletion of transcription factor E2-2 from mature peripheral PDC caused their spontaneous differentiation into cells with cDC properties. This included the loss of PDC markers, increase in MHC class II expression and T cell priming capacity, acquisition of dendritic morphology and induction of cDC signature genes. Genome-wide chromatin immunoprecipitation revealed direct binding of E2-2 to key PDC-specific and lymphoid genes, as well as to certain genes enriched in cDC. Thus, E2-2 actively maintains the cell fate of mature PDC and opposes the “default” cDC fate, in part through direct regulation of lineage-specific gene expres [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'The interferon-producing plasmacytoid dendritic cells (PDC) share common progenitors with antigen-presenting classical dendritic cells (cDC), yet they possess distinct morphology and molecular features resembling those of lymphocytes. It is unclear whether the unique cell fate of PDC is actively maintained in the steady state. We report that the deletion of transcription factor E2-2 from mature peripheral PDC caused their spontaneous differentiation into cells with cDC properties. This included the loss of PDC markers, increase in MHC class II expression and T cell priming capacity, acquisition of dendritic morphology and induction of cDC signature genes. Genome-wide chromatin immunoprecipitation revealed direct binding of E2-2 to key PDC-specific and lymphoid genes, as well as to certain genes enriched in cDC. Thus, E2-2 actively maintains the cell fate of mature PDC and opposes the “default” cDC fate, in part through direct regulation of lineage-specific gene expres [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string '“Memory-like T cells” are a subset of thymic cells that acquire effector function through the maturation process rather than interaction with specific antigen. Disruption of genes encoding T cell signaling proteins or transcription factors have provided insights into the differentiation of such cells. We show here that in BALB/c but not C57BL/6 mice, a large portion of thymic CD4-CD8+ T cells exhibit a memory-like phenotype. In BALB/c mice, IL-4 secreted by invariant natural killer T (iNKT) cells is both essential and sufficient for the generation of memory-like T cells. In C57BL/6 mice, iNKT cells are less abundant, producing IL-4 that is insufficient to induce thymic memory-like CD8+ T cells. BALB/c mice deficient in the transcription factor Kruppel-like factor (KLF) 13 have comparable numbers of iNKT cells to C57BL/6 mice and extremely low levels of thymic memory-like CD8+ T cells. This work documents the dramatic impact of a small number of KLF13-dependent iNKT ce [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string '“Memory-like T cells” are a subset of thymic cells that acquire effector function through the maturation process rather than interaction with specific antigen. Disruption of genes encoding T cell signaling proteins or transcription factors have provided insights into the differentiation of such cells. We show here that in BALB/c but not C57BL/6 mice, a large portion of thymic CD4-CD8+ T cells exhibit a memory-like phenotype. In BALB/c mice, IL-4 secreted by invariant natural killer T (iNKT) cells is both essential and sufficient for the generation of memory-like T cells. In C57BL/6 mice, iNKT cells are less abundant, producing IL-4 that is insufficient to induce thymic memory-like CD8+ T cells. BALB/c mice deficient in the transcription factor Kruppel-like factor (KLF) 13 have comparable numbers of iNKT cells to C57BL/6 mice and extremely low levels of thymic memory-like CD8+ T cells. This work documents the dramatic impact of a small number of KLF13-dependent iNKT ce [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'We used microarrays to detail the global programme of gene expression by circulating TCRVgamma9+ gamma delta T cells isolated from healthy individuals,tested either as resting cells or cells activated by phosphoantigen BrHPP and IL-2at an early(+6hrs) and a late (+7days) timepoint. We find that with more “NK cell” genes than alphabeta T cells and more “T cell” genes than NK cells, the circulating TCRVgamma9+ gamma delta T cells cells have a hybrid transcriptome. The gene signature of the activated cells recapitulates their physiological functions: Th1 cytokine, chemokine and cytotoxic activities at first and mitotic activity at later time points. The gene expression pattern of activated normal gamma delta T cells is nevertheless clearly distinctive from that of NK/T and peripheral T cell lymphomas of the gamma delta subtype.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'We used microarrays to detail the global programme of gene expression by circulating TCRVgamma9+ gamma delta T cells isolated from healthy individuals,tested either as resting cells or cells activated by phosphoantigen BrHPP and IL-2at an early(+6hrs) and a late (+7days) timepoint. We find that with more “NK cell” genes than alphabeta T cells and more “T cell” genes than NK cells, the circulating TCRVgamma9+ gamma delta T cells cells have a hybrid transcriptome. The gene signature of the activated cells recapitulates their physiological functions: Th1 cytokine, chemokine and cytotoxic activities at first and mitotic activity at later time points. The gene expression pattern of activated normal gamma delta T cells is nevertheless clearly distinctive from that of NK/T and peripheral T cell lymphomas of the gamma delta subtype.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'We used microarrays to detail the global programme of gene expression by circulating TCRVgamma9+ gamma delta T cells isolated from healthy individuals,tested either as resting cells or cells activated by phosphoantigen BrHPP and IL-2at an early(+6hrs) and a late (+7days) timepoint. We find that with more “NK cell” genes than alphabeta T cells and more “T cell” genes than NK cells, the circulating TCRVgamma9+ gamma delta T cells cells have a hybrid transcriptome. The gene signature of the activated cells recapitulates their physiological functions: Th1 cytokine, chemokine and cytotoxic activities at first and mitotic activity at later time points. The gene expression pattern of activated normal gamma delta T cells is nevertheless clearly distinctive from that of NK/T and peripheral T cell lymphomas of the gamma delta subtype.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'We used microarrays to detail the global programme of gene expression by circulating TCRVgamma9+ gamma delta T cells isolated from healthy individuals,tested either as resting cells or cells activated by phosphoantigen BrHPP and IL-2at an early(+6hrs) and a late (+7days) timepoint. We find that with more “NK cell” genes than alphabeta T cells and more “T cell” genes than NK cells, the circulating TCRVgamma9+ gamma delta T cells cells have a hybrid transcriptome. The gene signature of the activated cells recapitulates their physiological functions: Th1 cytokine, chemokine and cytotoxic activities at first and mitotic activity at later time points. The gene expression pattern of activated normal gamma delta T cells is nevertheless clearly distinctive from that of NK/T and peripheral T cell lymphomas of the gamma delta subtype.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'We used microarrays to detail the global programme of gene expression by circulating TCRVgamma9+ gamma delta T cells isolated from healthy individuals,tested either as resting cells or cells activated by phosphoantigen BrHPP and IL-2at an early(+6hrs) and a late (+7days) timepoint. We find that with more “NK cell” genes than alphabeta T cells and more “T cell” genes than NK cells, the circulating TCRVgamma9+ gamma delta T cells cells have a hybrid transcriptome. The gene signature of the activated cells recapitulates their physiological functions: Th1 cytokine, chemokine and cytotoxic activities at first and mitotic activity at later time points. The gene expression pattern of activated normal gamma delta T cells is nevertheless clearly distinctive from that of NK/T and peripheral T cell lymphomas of the gamma delta subtype.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'We used microarrays to detail the global programme of gene expression by circulating TCRVgamma9+ gamma delta T cells isolated from healthy individuals,tested either as resting cells or cells activated by phosphoantigen BrHPP and IL-2at an early(+6hrs) and a late (+7days) timepoint. We find that with more “NK cell” genes than alphabeta T cells and more “T cell” genes than NK cells, the circulating TCRVgamma9+ gamma delta T cells cells have a hybrid transcriptome. The gene signature of the activated cells recapitulates their physiological functions: Th1 cytokine, chemokine and cytotoxic activities at first and mitotic activity at later time points. The gene expression pattern of activated normal gamma delta T cells is nevertheless clearly distinctive from that of NK/T and peripheral T cell lymphomas of the gamma delta subtype.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Pathogen-associated molecular patterns decisively influence antiviral immune responses, whereas the contribution of endogenous signals of tissue damage, also known as “damage-associated molecular patterns” or “alarmins”, remains ill-defined. We show that interleukin-33 (IL-33), an alarmin released from necrotic cells, is necessary for potent CD8+ T cell (CTL) responses to replicating, prototypic RNA and DNA viruses in mice. IL-33 signaled through its receptor on activated CTLs, enhanced clonal expansion in a MyD88-dependent, CTL-intrinsic fashion, determined polyfunctional effector cell differentiation and was necessary for virus control. Moreover, recombinant IL-33 augmented vaccine-induced CTL responses. Radio-resistant cells of the splenic T cell zone produced IL-33, and efficient CTL responses required IL-33 from radio-resistant cells but not from hematopoietic cells. Thus, alarmin release by radio-resistant cells orchestrates protective antiviral CTL response [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Pathogen-associated molecular patterns decisively influence antiviral immune responses, whereas the contribution of endogenous signals of tissue damage, also known as “damage-associated molecular patterns” or “alarmins”, remains ill-defined. We show that interleukin-33 (IL-33), an alarmin released from necrotic cells, is necessary for potent CD8+ T cell (CTL) responses to replicating, prototypic RNA and DNA viruses in mice. IL-33 signaled through its receptor on activated CTLs, enhanced clonal expansion in a MyD88-dependent, CTL-intrinsic fashion, determined polyfunctional effector cell differentiation and was necessary for virus control. Moreover, recombinant IL-33 augmented vaccine-induced CTL responses. Radio-resistant cells of the splenic T cell zone produced IL-33, and efficient CTL responses required IL-33 from radio-resistant cells but not from hematopoietic cells. Thus, alarmin release by radio-resistant cells orchestrates protective antiviral CTL response [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Discrimination between self vs. non-self and adequate response to infection and tissue damage are fundamental functions of the immune system. The rapid and global spread of known and emerging viruses is a testament that the timely detection of viral pathogens that reproduce within host cells, presents a formidable challenge to the immune system. To gain access to a proper reproductive niche, many pathogens travel via the host vasculature and therefore become exposed to humoral factors of the innate immune system. Although a cascade of coagulation factors plays a fundamental role in host defense for “living fossils” such as horseshoe crabs (Xiphosurida spp), the role of the coagulation system in activation of innate responses to pathogens in higher organisms remains unclear. When human type C adenovirus (HAdv) enters the circulation, 240 copies of coagulation factor X (FX) bind to the virus particle with picomolar affinity. Here, using molecular dynamics flexible fitti [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Discrimination between self vs. non-self and adequate response to infection and tissue damage are fundamental functions of the immune system. The rapid and global spread of known and emerging viruses is a testament that the timely detection of viral pathogens that reproduce within host cells, presents a formidable challenge to the immune system. To gain access to a proper reproductive niche, many pathogens travel via the host vasculature and therefore become exposed to humoral factors of the innate immune system. Although a cascade of coagulation factors plays a fundamental role in host defense for “living fossils” such as horseshoe crabs (Xiphosurida spp), the role of the coagulation system in activation of innate responses to pathogens in higher organisms remains unclear. When human type C adenovirus (HAdv) enters the circulation, 240 copies of coagulation factor X (FX) bind to the virus particle with picomolar affinity. Here, using molecular dynamics flexible fitti [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Discrimination between self vs. non-self and adequate response to infection and tissue damage are fundamental functions of the immune system. The rapid and global spread of known and emerging viruses is a testament that the timely detection of viral pathogens that reproduce within host cells, presents a formidable challenge to the immune system. To gain access to a proper reproductive niche, many pathogens travel via the host vasculature and therefore become exposed to humoral factors of the innate immune system. Although a cascade of coagulation factors plays a fundamental role in host defense for “living fossils” such as horseshoe crabs (Xiphosurida spp), the role of the coagulation system in activation of innate responses to pathogens in higher organisms remains unclear. When human type C adenovirus (HAdv) enters the circulation, 240 copies of coagulation factor X (FX) bind to the virus particle with picomolar affinity. Here, using molecular dynamics flexible fitti [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Discrimination between self vs. non-self and adequate response to infection and tissue damage are fundamental functions of the immune system. The rapid and global spread of known and emerging viruses is a testament that the timely detection of viral pathogens that reproduce within host cells, presents a formidable challenge to the immune system. To gain access to a proper reproductive niche, many pathogens travel via the host vasculature and therefore become exposed to humoral factors of the innate immune system. Although a cascade of coagulation factors plays a fundamental role in host defense for “living fossils” such as horseshoe crabs (Xiphosurida spp), the role of the coagulation system in activation of innate responses to pathogens in higher organisms remains unclear. When human type C adenovirus (HAdv) enters the circulation, 240 copies of coagulation factor X (FX) bind to the virus particle with picomolar affinity. Here, using molecular dynamics flexible fitti [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Discrimination between self vs. non-self and adequate response to infection and tissue damage are fundamental functions of the immune system. The rapid and global spread of known and emerging viruses is a testament that the timely detection of viral pathogens that reproduce within host cells, presents a formidable challenge to the immune system. To gain access to a proper reproductive niche, many pathogens travel via the host vasculature and therefore become exposed to humoral factors of the innate immune system. Although a cascade of coagulation factors plays a fundamental role in host defense for “living fossils” such as horseshoe crabs (Xiphosurida spp), the role of the coagulation system in activation of innate responses to pathogens in higher organisms remains unclear. When human type C adenovirus (HAdv) enters the circulation, 240 copies of coagulation factor X (FX) bind to the virus particle with picomolar affinity. Here, using molecular dynamics flexible fitti [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Discrimination between self vs. non-self and adequate response to infection and tissue damage are fundamental functions of the immune system. The rapid and global spread of known and emerging viruses is a testament that the timely detection of viral pathogens that reproduce within host cells, presents a formidable challenge to the immune system. To gain access to a proper reproductive niche, many pathogens travel via the host vasculature and therefore become exposed to humoral factors of the innate immune system. Although a cascade of coagulation factors plays a fundamental role in host defense for “living fossils” such as horseshoe crabs (Xiphosurida spp), the role of the coagulation system in activation of innate responses to pathogens in higher organisms remains unclear. When human type C adenovirus (HAdv) enters the circulation, 240 copies of coagulation factor X (FX) bind to the virus particle with picomolar affinity. Here, using molecular dynamics flexible fitti [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Discrimination between self vs. non-self and adequate response to infection and tissue damage are fundamental functions of the immune system. The rapid and global spread of known and emerging viruses is a testament that the timely detection of viral pathogens that reproduce within host cells, presents a formidable challenge to the immune system. To gain access to a proper reproductive niche, many pathogens travel via the host vasculature and therefore become exposed to humoral factors of the innate immune system. Although a cascade of coagulation factors plays a fundamental role in host defense for “living fossils” such as horseshoe crabs (Xiphosurida spp), the role of the coagulation system in activation of innate responses to pathogens in higher organisms remains unclear. When human type C adenovirus (HAdv) enters the circulation, 240 copies of coagulation factor X (FX) bind to the virus particle with picomolar affinity. Here, using molecular dynamics flexible fitti [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Discrimination between self vs. non-self and adequate response to infection and tissue damage are fundamental functions of the immune system. The rapid and global spread of known and emerging viruses is a testament that the timely detection of viral pathogens that reproduce within host cells, presents a formidable challenge to the immune system. To gain access to a proper reproductive niche, many pathogens travel via the host vasculature and therefore become exposed to humoral factors of the innate immune system. Although a cascade of coagulation factors plays a fundamental role in host defense for “living fossils” such as horseshoe crabs (Xiphosurida spp), the role of the coagulation system in activation of innate responses to pathogens in higher organisms remains unclear. When human type C adenovirus (HAdv) enters the circulation, 240 copies of coagulation factor X (FX) bind to the virus particle with picomolar affinity. Here, using molecular dynamics flexible fitti [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Discrimination between self vs. non-self and adequate response to infection and tissue damage are fundamental functions of the immune system. The rapid and global spread of known and emerging viruses is a testament that the timely detection of viral pathogens that reproduce within host cells, presents a formidable challenge to the immune system. To gain access to a proper reproductive niche, many pathogens travel via the host vasculature and therefore become exposed to humoral factors of the innate immune system. Although a cascade of coagulation factors plays a fundamental role in host defense for “living fossils” such as horseshoe crabs (Xiphosurida spp), the role of the coagulation system in activation of innate responses to pathogens in higher organisms remains unclear. When human type C adenovirus (HAdv) enters the circulation, 240 copies of coagulation factor X (FX) bind to the virus particle with picomolar affinity. Here, using molecular dynamics flexible fitti [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Discrimination between self vs. non-self and adequate response to infection and tissue damage are fundamental functions of the immune system. The rapid and global spread of known and emerging viruses is a testament that the timely detection of viral pathogens that reproduce within host cells, presents a formidable challenge to the immune system. To gain access to a proper reproductive niche, many pathogens travel via the host vasculature and therefore become exposed to humoral factors of the innate immune system. Although a cascade of coagulation factors plays a fundamental role in host defense for “living fossils” such as horseshoe crabs (Xiphosurida spp), the role of the coagulation system in activation of innate responses to pathogens in higher organisms remains unclear. When human type C adenovirus (HAdv) enters the circulation, 240 copies of coagulation factor X (FX) bind to the virus particle with picomolar affinity. Here, using molecular dynamics flexible fitti [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Discrimination between self vs. non-self and adequate response to infection and tissue damage are fundamental functions of the immune system. The rapid and global spread of known and emerging viruses is a testament that the timely detection of viral pathogens that reproduce within host cells, presents a formidable challenge to the immune system. To gain access to a proper reproductive niche, many pathogens travel via the host vasculature and therefore become exposed to humoral factors of the innate immune system. Although a cascade of coagulation factors plays a fundamental role in host defense for “living fossils” such as horseshoe crabs (Xiphosurida spp), the role of the coagulation system in activation of innate responses to pathogens in higher organisms remains unclear. When human type C adenovirus (HAdv) enters the circulation, 240 copies of coagulation factor X (FX) bind to the virus particle with picomolar affinity. Here, using molecular dynamics flexible fitti [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Discrimination between self vs. non-self and adequate response to infection and tissue damage are fundamental functions of the immune system. The rapid and global spread of known and emerging viruses is a testament that the timely detection of viral pathogens that reproduce within host cells, presents a formidable challenge to the immune system. To gain access to a proper reproductive niche, many pathogens travel via the host vasculature and therefore become exposed to humoral factors of the innate immune system. Although a cascade of coagulation factors plays a fundamental role in host defense for “living fossils” such as horseshoe crabs (Xiphosurida spp), the role of the coagulation system in activation of innate responses to pathogens in higher organisms remains unclear. When human type C adenovirus (HAdv) enters the circulation, 240 copies of coagulation factor X (FX) bind to the virus particle with picomolar affinity. Here, using molecular dynamics flexible fitti [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Discrimination between self vs. non-self and adequate response to infection and tissue damage are fundamental functions of the immune system. The rapid and global spread of known and emerging viruses is a testament that the timely detection of viral pathogens that reproduce within host cells, presents a formidable challenge to the immune system. To gain access to a proper reproductive niche, many pathogens travel via the host vasculature and therefore become exposed to humoral factors of the innate immune system. Although a cascade of coagulation factors plays a fundamental role in host defense for “living fossils” such as horseshoe crabs (Xiphosurida spp), the role of the coagulation system in activation of innate responses to pathogens in higher organisms remains unclear. When human type C adenovirus (HAdv) enters the circulation, 240 copies of coagulation factor X (FX) bind to the virus particle with picomolar affinity. Here, using molecular dynamics flexible fitti [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Discrimination between self vs. non-self and adequate response to infection and tissue damage are fundamental functions of the immune system. The rapid and global spread of known and emerging viruses is a testament that the timely detection of viral pathogens that reproduce within host cells, presents a formidable challenge to the immune system. To gain access to a proper reproductive niche, many pathogens travel via the host vasculature and therefore become exposed to humoral factors of the innate immune system. Although a cascade of coagulation factors plays a fundamental role in host defense for “living fossils” such as horseshoe crabs (Xiphosurida spp), the role of the coagulation system in activation of innate responses to pathogens in higher organisms remains unclear. When human type C adenovirus (HAdv) enters the circulation, 240 copies of coagulation factor X (FX) bind to the virus particle with picomolar affinity. Here, using molecular dynamics flexible fitti [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Regulatory T (Treg) cells, whose identity and function are defined by the transcription factor Foxp3, are indispensable for immune homeostasis. It is unclear whether Foxp3 exerts its Treg lineage specification function through active modification of the chromatin landscape and establishment of new enhancers or by exploiting a pre-existing enhancer landscape. Analysis of the chromatin accessibility of Foxp3-bound enhancers in Treg and Foxp3-negative T cells showed that Foxp3 was bound overwhelmingly to pre-accessible enhancers occupied by its cofactors in precursor cells or a structurally related predecessor. Furthermore, the bulk of Foxp3-bound Treg cell enhancers inaccessible in Foxp3- CD4+ cells became accessible upon T cell receptor activation prior to Foxp3 expression with only a small subset associated with several functionally important genes being exclusively Treg cell-specific. Thus, in a late cellular differentiation process Foxp3 defines Treg cell functionality  [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Regulatory T (Treg) cells, whose identity and function are defined by the transcription factor Foxp3, are indispensable for immune homeostasis. It is unclear whether Foxp3 exerts its Treg lineage specification function through active modification of the chromatin landscape and establishment of new enhancers or by exploiting a pre-existing enhancer landscape. Analysis of the chromatin accessibility of Foxp3-bound enhancers in Treg and Foxp3-negative T cells showed that Foxp3 was bound overwhelmingly to pre-accessible enhancers occupied by its cofactors in precursor cells or a structurally related predecessor. Furthermore, the bulk of Foxp3-bound Treg cell enhancers inaccessible in Foxp3- CD4+ cells became accessible upon T cell receptor activation prior to Foxp3 expression with only a small subset associated with several functionally important genes being exclusively Treg cell-specific. Thus, in a late cellular differentiation process Foxp3 defines Treg cell functionality  [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Regulatory T (Treg) cells, whose identity and function are defined by the transcription factor Foxp3, are indispensable for immune homeostasis. It is unclear whether Foxp3 exerts its Treg lineage specification function through active modification of the chromatin landscape and establishment of new enhancers or by exploiting a pre-existing enhancer landscape. Analysis of the chromatin accessibility of Foxp3-bound enhancers in Treg and Foxp3-negative T cells showed that Foxp3 was bound overwhelmingly to pre-accessible enhancers occupied by its cofactors in precursor cells or a structurally related predecessor. Furthermore, the bulk of Foxp3-bound Treg cell enhancers inaccessible in Foxp3- CD4+ cells became accessible upon T cell receptor activation prior to Foxp3 expression with only a small subset associated with several functionally important genes being exclusively Treg cell-specific. Thus, in a late cellular differentiation process Foxp3 defines Treg cell functionality  [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Regulatory T (Treg) cells, whose identity and function are defined by the transcription factor Foxp3, are indispensable for immune homeostasis. It is unclear whether Foxp3 exerts its Treg lineage specification function through active modification of the chromatin landscape and establishment of new enhancers or by exploiting a pre-existing enhancer landscape. Analysis of the chromatin accessibility of Foxp3-bound enhancers in Treg and Foxp3-negative T cells showed that Foxp3 was bound overwhelmingly to pre-accessible enhancers occupied by its cofactors in precursor cells or a structurally related predecessor. Furthermore, the bulk of Foxp3-bound Treg cell enhancers inaccessible in Foxp3- CD4+ cells became accessible upon T cell receptor activation prior to Foxp3 expression with only a small subset associated with several functionally important genes being exclusively Treg cell-specific. Thus, in a late cellular differentiation process Foxp3 defines Treg cell functionality  [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Regulatory T (Treg) cells, whose identity and function are defined by the transcription factor Foxp3, are indispensable for immune homeostasis. It is unclear whether Foxp3 exerts its Treg lineage specification function through active modification of the chromatin landscape and establishment of new enhancers or by exploiting a pre-existing enhancer landscape. Analysis of the chromatin accessibility of Foxp3-bound enhancers in Treg and Foxp3-negative T cells showed that Foxp3 was bound overwhelmingly to pre-accessible enhancers occupied by its cofactors in precursor cells or a structurally related predecessor. Furthermore, the bulk of Foxp3-bound Treg cell enhancers inaccessible in Foxp3- CD4+ cells became accessible upon T cell receptor activation prior to Foxp3 expression with only a small subset associated with several functionally important genes being exclusively Treg cell-specific. Thus, in a late cellular differentiation process Foxp3 defines Treg cell functionality  [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'Regulatory T (Treg) cells, whose identity and function are defined by the transcription factor Foxp3, are indispensable for immune homeostasis. It is unclear whether Foxp3 exerts its Treg lineage specification function through active modification of the chromatin landscape and establishment of new enhancers or by exploiting a pre-existing enhancer landscape. Analysis of the chromatin accessibility of Foxp3-bound enhancers in Treg and Foxp3-negative T cells showed that Foxp3 was bound overwhelmingly to pre-accessible enhancers occupied by its cofactors in precursor cells or a structurally related predecessor. Furthermore, the bulk of Foxp3-bound Treg cell enhancers inaccessible in Foxp3- CD4+ cells became accessible upon T cell receptor activation prior to Foxp3 expression with only a small subset associated with several functionally important genes being exclusively Treg cell-specific. Thus, in a late cellular differentiation process Foxp3 defines Treg cell functionality  [... truncated]</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>input string 'During acute viral infections, naïve CD8+ T cells differentiate into effector CD8+ T cells and, after viral control, into memory CD8+ T cells. Memory CD8+ T cells are highly functional, proliferate rapidly upon reinfection and persist long-term without antigen. In contrast, during chronic infections, CD8+ T cells become “exhausted” and have poor effector function, express multiple inhibitory receptors, possess low proliferative capacity, and cannot persist without antigen. To compare the development of functional memory T cells with poorly functional exhausted T cells, we generated longitudinal transcriptional profiles for each.' cannot be translated to UTF-8, is it valid in 'CP1252'?</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>Assuming the organism to be human.</span>
<span class="r-in"><span class="fu"><a href="plotMsigNetwork.html">plotMsigNetwork</a></span><span class="op">(</span><span class="va">ig</span><span class="op">)</span></span>
<span class="r-plt"><img src="characteriseGeneset-1.png" alt="" width="700" height="433"></span>
<span class="r-in"><span class="co"># }</span></span>
<span class="r-in"></span>
</code></pre></div>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top"><h2 data-toc-skip>Contents</h2>
    </nav></div>
</div>


      <footer><div class="copyright">
  <p></p><p>Developed by Dharmesh D. Bhuva.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 1.9000.9000.9000.</p>
</div>

      </footer></div>

  


  

  </body></html>

